src/libraries/System.Private.CoreLib/src/System/Lazy.cs

```diff
@@ -430,7 +430,7 @@ private T CreateValue()
                         break;
                 }
             }
-            return Value;
+            return Value!;
```

Comment:
>Value! [](start = 19, length = 6)

Is the `!` necessary?

Reply:
Nope, thanks, it was left-over from an intermediate stage.